addons/l10n_fr_account/models/template_fr_pca.py

```diff
@@ -0,0 +1,64 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from odoo import models, _, Command
+from odoo.addons.account.models.chart_template import template
+
+
+class AccountChartTemplate(models.AbstractModel):
+    _inherit = 'account.chart.template'
+
+    @template('fr_pca')
+    def _get_fr_pca_template_data(self):
+        return {
+            'name': _('Plan comptable associatif'),
```

Comment:
Same as my comment above: the text to translate should be in English (and be correctly translated in French through the fr.po file)